src/app/views/query-runner/QueryRunner.tsx

```diff
@@ -22,6 +22,10 @@ export class QueryRunner extends Component<IQueryRunnerProps, IQueryRunnerState>
       ],
       selectedVerb: 'GET',
       sampleURL: 'https://graph.microsoft.com/v1.0/me/',
+      sampleBody: undefined,
```

Comment:
can we use `null` or `''` instead of `undefined`  so that it can be easy to debug.

Reply:
I prefer an empty string.